### PR TITLE
Add tooling suffix property

### PR DESCRIPTION
**Pull request by ankitkumarr**

### CODE REVIEW

**Comment by ankitkumarr** [comment]
@phil-allen-msft

File: cli-feed-v4.json [hunk 1]
```diff
@@ -627,6 +709,123 @@
...
+              "targetFramework": ".NET 5",
+              "description": "Isolated"
+            },
+            "capabilities": "isolated,net5",
+            "sdk": {
+              "name": "Microsoft.Azure.Functions.Worker.Sdk",
+              "version": "1.0.1"
+            },
+            "default": false,
+            "toolingSuffix": "net5-isolated",
```
Comment (ankitkumarr) [inline cli-feed-v4.json:765]: This is the new property

File: cli-feed-v4.json [hunk 2]
```diff
@@ -627,6 +709,123 @@
...
+            },
+            "capabilities": "",
+            "sdk": {
+              "name": "Microsoft.NET.Sdk.Functions",
+              "version": "3.0.11"
+            },
+            "default": true,
+            "localEntryPoint": "func.exe",
+            "targetFramework": "netcoreapp3.1",
+            "toolingSuffix": "",
```
Comment (ankitkumarr) [inline cli-feed-v4.json:733]: empty for .NET Core 3 inproc

Comment (soninaren) [reply cli-feed-v4.json:733]: We should opt for not having the property instead of empty string.

Comment (ankitkumarr) [reply cli-feed-v4.json:733]: I think @phil-allen-msft suggested to define empty. Phil, do you prefer if we don't have the property at all?

Comment (ankitkumarr) [reply cli-feed-v4.json:733]: Thank you! I will remove this property from .NET Core 3 entries then.

**Review by madelinegordon — APPROVED** [review]

**Pull request merged**